tests/test_maker_utils.py

```diff
@@ -1,190 +0,0 @@
-import inspect
-from unittest import mock
-
-import asdf
-import gwcs
-import pytest
-from astropy import units as u
-from astropy.time import Time
-
-import roman_datamodels._maker_utils as maker_utils
-from roman_datamodels import datamodels, stnode
-from roman_datamodels.datamodels._datamodels import _RomanDataModel
-from roman_datamodels.testing import assert_node_equal
-
-
-def test_deprecation():
-    with pytest.warns(DeprecationWarning, match="maker_utils is deprecated"):
-        import roman_datamodels.maker_utils  # noqa
-
-
-@pytest.mark.parametrize("node_class", stnode.NODE_CLASSES)
-def test_maker_utility_implemented(node_class, request):
```

Comment:
Is there a test for the `create_*` methods to smoke out that those "function" for all of those classes?

Reply:
Here's the test for `create_fake_data` for all models:
https://github.com/spacetelescope/roman_datamodels/blob/005e4a7b0cf4efced9291ae8011fc3b853a50a43/tests/test_models.py#L774-L779